clientv3/client.go

```diff
@@ -44,8 +44,18 @@ import (
 var (
 	ErrNoAvailableEndpoints = errors.New("etcdclient: no available endpoints")
 	ErrOldCluster           = errors.New("etcdclient: old cluster version")
+
+	defaultBalancer balancer.Balancer
 )
 
+func init() {
+	defaultBalancer = balancer.New(balancer.Config{
+		Policy: picker.RoundrobinBalanced,
+		Name:   "rrbalancer",
```

Comment:
`Name: picker.RoundrobinBalanced.String(),` or define `rrbalancer` as `const`?

Reply:
Since we're registering a grpc globally in a process where the etcd client might not be the only thing grpc is used for, I'll scope it-- `fmt.Sprintf("etcd-%s", picker.RoundrobinBalanced.String())`.